faker/providers/ssn/cs_CZ/__init__.py

```diff
@@ -8,10 +8,36 @@ class Provider(BaseProvider):
         'CZ##########',
     )
 
+    national_id_months = ['%.2d' % i for i in range(1, 13)] + ['%.2d' % i for i in range(51, 63)]
+
     def vat_id(self):
         """
         http://ec.europa.eu/taxation_customs/vies/faq.html#item_11
         :return: A random Czech VAT ID
         """
 
         return self.bothify(self.random_element(self.vat_id_formats))
+
+    def birth_number(self):
+        """
+        Birth Number (Czech/Slovak: rodné číslo (RČ))
+        https://en.wikipedia.org/wiki/National_identification_number#Czech_Republic_and_Slovakia
+        """
+        birthdate = self.generator.date_of_birth()
+        year = '%.2d' % (birthdate.year % 100)
+        month = self.random_element(self.national_id_months)
+        day = '%.2d' % birthdate.day
+        if birthdate.year > 1953:
+            sn = self.random_number(4, True)
+        else:
+            sn = self.random_number(3, True)
+        number = int('{}{}{}{}'.format(year, month, day, sn))
+        while number % 11 != 0:
+            number += 1
```

Comment:
instead of iterating, wouldn't be more efficient to calculate this as `number = 11 * (number % 11 + 1)`?

Reply:
Good point, there isn't a need to iterate here. How's `math.ceil(number / 11) * 11`? I've pushed up a commit with that in place of the iteration.